Jenkinsfile

```diff
@@ -144,6 +162,12 @@ pipeline {
                         }
                     }
                 }
+
+                stage('Publish test results') {
+                    steps {
+                        junit 'test/reports/*.xml'
```

Comment:
maybe try https://stackoverflow.com/questions/67162746/how-to-get-rid-of-noisy-warning-no-suitable-checks-publisher-found to make this stage green?